README.md

```diff
@@ -1,12 +1,15 @@
-# React + Vite
+# TO-DO APPLICATION BY VERONICA FERNANDEZ-DIAZ
 
-This template provides a minimal setup to get React working in Vite with HMR and some ESLint rules.
+## Project Description
 
-Currently, two official plugins are available:
+This project will demonstrate how to build a To-Do application for lists of tasks.
 
-- [@vitejs/plugin-react](https://github.com/vitejs/vite-plugin-react/blob/main/packages/plugin-react/README.md) uses [Babel](https://babeljs.io/) for Fast Refresh
-- [@vitejs/plugin-react-swc](https://github.com/vitejs/vite-plugin-react-swc) uses [SWC](https://swc.rs/) for Fast Refresh
+## Installation Process
 
-## Expanding the ESLint configuration
-
-If you are developing a production application, we recommend using TypeScript and enable type-aware lint rules. Check out the [TS template](https://github.com/vitejs/vite/tree/main/packages/create-vite/template-react-ts) to integrate TypeScript and [`typescript-eslint`](https://typescript-eslint.io) in your project.
+1. Choose a built tool and server such as using Vite with a React template
```

Comment:
Consider what these steps are directing. What you want to do is provide exact instructions for someone to follow, which will allow them to run your project on their machine. Not how to create a Vite template.